code/modules/projectiles/gun_attachables.dm

```diff
@@ -1061,6 +1061,20 @@ Defined in conflicts.dm of the #defines folder.
 	scatter_mod = 0
 	movement_acc_penalty_mod = 0
 
+/obj/item/attachable/stock/t60stock
+	name = "T-60 Stock"
```

Comment:
I'm kinda a little annoyed by this, maybe make the generic (such as Rifle, Machine Gun, etc and specifically, "Stock") names into lowercase instead of the first letter being uppercase? To maintain consistency?

IE, make it into "stock" instead of "Stock"?